ansys/mapdl/core/_commands/database/selecting.py

```diff
@@ -130,6 +130,43 @@ def asel(
 
             STAT - Display the current select status.
 
+
+        The following fields are used only with Type = S, R, A, or U:
+
+        Item
+            Label identifying data. Valid item labels are shown in Table 105: ASEL - Valid Item and Component
+            Labels (p. 185). Some items also require a component label. If Item = PICK (or simply "P"), graphical
+            picking is enabled and all remaining command fields are ignored (valid only in the GUI). Defaults
+            to AREA.
+
+        Comp
+            Component of the item (if required). Valid component labels are shown in Table 105: ASEL - Valid
+            Item and Component Labels (p. 185).
+
+        VMIN
+            Minimum value of item range. Ranges are area numbers, coordinate values, attribute numbers, etc.,
+            as appropriate for the item. A component name (as specified on the CM (p. 338) command) may
+            also be substituted for VMIN (VMAX and VINC are ignored). If Item = MAT, TYPE, REAL, or ESYS
+            and if VMIN is positive, the absolute value of Item is compared against the range for selection; if
+            VMIN is negative, the signed value of Item is compared. See the ALIST (p. 106) command for a
+            discussion of signed attributes.
+
+        VMAX
+            Maximum value of item range. VMAX defaults to VMIN.
+
+        VINC
+            Value increment within range. Used only with integer ranges (such as for area numbers). Defaults
+            to 1. VINC cannot be negative.
+
+        KSWP
+            Specifies whether only areas are to be selected:
+
+                - `kswp = 0` - Select areas only.
```

Comment:
Note for the future: Use double back tick instead of single. Double back tick in rst gets rendered as literal, while single gets rendered as italic.

Reply:
I will try to remember.